tests/integration/compose.yml

```diff
@@ -0,0 +1,23 @@
+services:
+  mosquitto:
+    container_name: mosquitto
+    image: eclipse-mosquitto:2.0
+    volumes:
+      - "./localdocker/mosquitto:/mosquitto"
+    ports:
+      - "1883:1883"
+    command: "mosquitto -v -c /mosquitto/mosquitto.conf"
+    restart: unless-stopped
+
+  virtuale3:
+    container_name: virtuale3
+    image: ghcr.io/lgraf/vituale3:sha-9bfe61b
```

Comment:
Aktuell wird ein von @MyHomeMyData patched virtual3e verwendet ([branch](https://github.com/MyHomeMyData/vitualE3/commit/8f8e823eab8165dfbe476dfbe3323a4a16b8ac0e)). Das Docker Image wird durch einen minimalistischen docker build workflow in meinem [fork](https://github.com/lgraf/vitualE3/commits/docker-build/) veröffentlicht.